File: pinyougou/pinyougou-manage-web/src/main/webapp/js/service/specificationService.js
app.service('specificationService',function ($http) {

    //查询单个信息
    this.findOne = function (id) {
        return $http.get('../specification/findOne.do?id='+id);
    }

    // 分页查询
    this.findPage = function (page, rows) {
        return $http.get("../specification/findPage.do?page=" + page + "&rows=" +rows);
    }

    //搜索
    this.search = function (page,rows,searchEntity) {
        return $http.post('../specification/search.do?page='+page+'&rows='+rows,searchEntity);
    }

    //添加
    this.add = function (entity) {
        return $http.post('../specification/add.do',entity)
    }

    //修改
    this.update = function (entity) {
        return $http.post('../specification/update.do',entity)
    }

    //批量删除
    this.delete = function (selectedIds) {
        return $http.get('../specification/delete.do?ids='+selectedIds);
    }
});

File: pinyougou/pinyougou-sellergoods/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSpecification;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.PageResult;
import com.pinyougou.vo.Specification;

public interface SpecificationService extends BaseService<TbSpecification>{

    public PageResult search(TbSpecification specification, Integer page, Integer rows);

    public void add(Specification specification);

    public Specification findOne(Long id);

    public void update(Specification specification);

    public void delete(Long[] ids);

}



